configure

```diff
@@ -16,10 +16,11 @@ fi
 ## Parse the args, only looking for -camldir
 ## We avoid using shift to keep "$@" intact
 
+cmd=$ocaml
 last=
 for i; do
    case $last in
-       -camldir|--camldir) cmd="$i/ocaml"; break;;
```

Comment:
I realized this just now but the output of `coqtop -help` shows that this was far from the only option to accept two dashes, so this doesn't really make things more uniform. Not a big deal.